### DISCUSSION TITLE
Neues SSL Zerifikat wird nach hochladen nicht benutzt

### DISCUSSION THREAD

**Discussion by jogicom** [OpenCCU/OpenCCU#2379] [🇩🇪 Fragen&Antworten] [answered]
Hallo zusammen,

ich wollte heute ein neues SSL Zertifikat in meine RaspberryMatic hochladen (3.69.7.20230626), aus unerfindlichen Gründen wird dies aber nicht akzeptiert. Bei dem Zertifikat handelt es sich um ein LetsEncrypt Zertifikat, das auf einer Synology Diskstation generiert wird.

Nach Export kopiere ich in folgender Reihenfolge den privateKey, cert und chain in eine Datei, diese lade ich dann in der RaspberyyMatic hoch. Der Webserver wird danach automatisch gestartet. Danach wird aber immer noch das alte Zertifikat benutzt.

Habe auch schon die chain weggelassen, ohne Erfolg. Als NotNagel habe ich auch schon versucht mit unterschiedlichen Zeilenenden (LF, LF+CR) zu arbeiten, Ergebnis bleibt aber leider immer das gleiche.

Ich habe schon zigMal die Zertifikate erneuert, hat bis jetzt immer geklappt.

Nun habe ich das bestehende Zertifikat, welches in 13 Tagen abläuft, von der RM gelöscht, die RM hat deshalb ein selfSigned Zertifikat erstellt, dieses wird dann nach dem Neustart des WebServers auch verwendet.

Ein Versuch ein neues Zertifikat hochzuladen schlägt weiterhin fehlt, wie beschrieben wird der Webserver neu gestartet und danach NICHT das neue Zertifikat verwendet.

Habt Ihr mir irgendwelche Tips ?

Hier das Log, umd 15:46:58h habe ich das bestehende Zertifikat gelöscht 
```
Aug  1 15:45:35 homematic user.info monit[1987]: 'lighttpd' restart on user request
Aug  1 15:45:35 homematic user.info monit[1987]: Monit daemon with PID 1987 awakened
Aug  1 15:45:35 homematic user.info monit[1987]: Awakened by User defined signal 1
Aug  1 15:45:35 homematic user.info monit[1987]: 'lighttpd' trying to restart
Aug  1 15:45:35 homematic user.info monit[1987]: 'lighttpd' restart: '/etc/init.d/S50lighttpd restart'
Aug  1 15:45:36 homematic user.info monit[1987]: 'lighttpd' restart action done
Aug  1 15:46:12 homematic user.info monit[1987]: 'lighttpd' restart on user request
Aug  1 15:46:12 homematic user.info monit[1987]: Monit daemon with PID 1987 awakened
Aug  1 15:46:17 homematic user.info monit[1987]: 'lighttpd' trying to restart
Aug  1 15:46:17 homematic user.info monit[1987]: 'lighttpd' restart: '/etc/init.d/S50lighttpd restart'
Aug  1 15:46:17 homematic user.info monit[1987]: 'lighttpd' restart action done
Aug  1 15:46:17 homematic user.info monit[1987]: Awakened by User defined signal 1
Aug  1 15:46:58 homematic user.info monit[1987]: 'lighttpd' certificate valid days test succeeded [valid for 3649 days]
Aug  1 15:49:18 homematic user.info monit[1987]: 'lighttpd' restart on user request
Aug  1 15:49:18 homematic user.info monit[1987]: Monit daemon with PID 1987 awakened
Aug  1 15:49:22 homematic user.info monit[1987]: 'lighttpd' trying to restart
Aug  1 15:49:22 homematic user.info monit[1987]: 'lighttpd' restart: '/etc/init.d/S50lighttpd restart'
Aug  1 15:49:23 homematic user.info monit[1987]: 'lighttpd' restart action done
Aug  1 15:49:23 homematic user.info monit[1987]: Awakened by User defined signal 1
Aug  1 15:59:29 homematic user.info monit[1987]: 'lighttpd' restart on user request
Aug  1 15:59:29 homematic user.info monit[1987]: Monit daemon with PID 1987 awakened
Aug  1 15:59:29 homematic user.info monit[1987]: Awakened by User defined signal 1
Aug  1 15:59:29 homematic user.info monit[1987]: 'lighttpd' trying to restart
Aug  1 15:59:29 homematic user.info monit[1987]: 'lighttpd' restart: '/etc/init.d/S50lighttpd restart'
Aug  1 15:59:30 homematic user.info monit[1987]: 'lighttpd' restart action done
```
Und hier der Versuch das neue Zertifikat hochzuladen
```
Aug  1 16:14:08 homematic user.info monit[1987]: 'lighttpd' restart on user request
Aug  1 16:14:08 homematic user.info monit[1987]: Monit daemon with PID 1987 awakened
Aug  1 16:14:08 homematic user.info monit[1987]: Awakened by User defined signal 1
Aug  1 16:14:08 homematic user.info monit[1987]: 'lighttpd' trying to restart
Aug  1 16:14:08 homematic user.info monit[1987]: 'lighttpd' restart: '/etc/init.d/S50lighttpd restart'
Aug  1 16:14:09 homematic user.info monit[1987]: 'lighttpd' restart action done
```
Der einzige Unterschied den ich sehe, dass kein `lighttpd' certificate valid days test succeeded [valid for xxxx days]` im Log erscheint.

**Comment by Baxxy13** [comment, marked as answer]
Weiß jetzt nicht ob es das gleiche Problem ist/war, aber schau dir mal #2322 an. 
In den Nightly Snapshots hatte Jens da was gefixt.

**Reply by jogicom** [reply]
Volltreffer! Danke
